Houseclub/src/main/java/me/grishka/houseclub/fragments/HomeFragment.java

```diff
@@ -113,20 +113,34 @@ public boolean wantsLightStatusBar(){
 
 	@Override
 	public void onCreateOptionsMenu(Menu menu, MenuInflater inflater){
-		menu.add(0,0,0,"").setIcon(R.drawable.ic_notifications).setShowAsAction(MenuItem.SHOW_AS_ACTION_ALWAYS);
-		menu.add(0,1,0,"").setIcon(R.drawable.ic_baseline_person_24).setShowAsAction(MenuItem.SHOW_AS_ACTION_ALWAYS);
+		inflater.inflate(R.menu.menu_home, menu);
 	}
 
 	@Override
 	public boolean onOptionsItemSelected(MenuItem item){
-		Bundle args=new Bundle();
-		args.putInt("id", Integer.parseInt(ClubhouseSession.userID));
-		if(item.getItemId()==0) {
-			Nav.go(getActivity(), NotificationListFragment.class, args);
-		} else if(item.getItemId()==1){
+		if (item.getItemId() == R.id.homeMenuProfile) {
+			Bundle args=new Bundle();
+			args.putInt("id", Integer.parseInt(ClubhouseSession.userID));
 			Nav.go(getActivity(), ProfileFragment.class, args);
+			return true;
+		} else if (item.getItemId() == R.id.homeMenuSearchPeople) {
+			Bundle args = new Bundle();
+			//args.putInt(BaseSearchFragment.KEY_SEARCH_TYPE, BaseSearchFragment.SearchType.PEOPLE.ordinal());
+			//Nav.go(getActivity(), SearchPeopleFragment.class, args);
+			Nav.go(getActivity(), SearchListFragment.class, args);
+			return true;
+		} else if (item.getItemId() == R.id.homeMenuNotifications) {
+			Bundle args = new Bundle();
+			args.putInt("id", Integer.parseInt(ClubhouseSession.userID));
+			Nav.go(getActivity(), NotificationListFragment.class, args);
+			return true;
+		} else if(item.getItemId() == R.id.homeMenuInvite) {
+
+			Bundle args = new Bundle();
+			Nav.go(getActivity(), InviteListFragment.class, args);
+
 		}
-		return true;
+		return false;
```

Comment:
Why?

Reply:
I'm changed this return to false independently too. Dunno why, but got some exceptions in navigation when more than 2 items in a menu...

Reply:
fixed

Reply:
@bit-world: with true here I had the same problem as @josefjebavy described, couldnt navigate twice to the same page.